geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/processors/package.scala

```diff
@@ -0,0 +1,76 @@
+/***********************************************************************
+ * Copyright (c) 2015-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.geomesa.nifi
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.nifi.components.PropertyDescriptor
+import org.apache.nifi.expression.ExpressionLanguageScope
+import org.apache.nifi.processor.util.StandardValidators
+import org.geotools.data.DataAccessFactory.Param
+import org.geotools.data.Parameter
+import org.locationtech.geomesa.utils.geotools.GeoMesaParam
+
+import scala.util.control.NonFatal
+
+package object processors extends LazyLogging {
+
+  /**
+    * Creates a nifi property descriptor based on a geotools data store parameter
+    *
+    * @param param param
+    * @return
+    */
+  def property(param: Param): PropertyDescriptor = {
```

Comment:
I'm not crazy about the name of this function.  I can live with it.  Seems like it could be something like 'createNiFiProperty' or paramToProperty?

Reply:
i can update it, that's what it was before

Reply:
If that's what it was before, no worries.

Reply:
renamed